docs/source/federation/hosted-gateway-definitions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hosted gateway configurations
+description: Managing gateways at scale
+---
+
+In many cases it is sufficient to define the list of services comprising a federated graph directly within the gateway:
+```javascript
+const gateway = await createGateway({
+    serviceList: [
+      { name: "accounts", url: "http://localhost:4001/graphql" },
+      { name: "products", url: "http://localhost:4002/graphql" },
+      ...
+    ]
+  });
+```
+
+However for larger scale projects it can be helpful to remotely manage service definitions, such that new services may be added or existing services modified at runtime. For these use cases, aApollo Gateway can be configured to pull a remotely hosted service definition from Apollo Engine.
+
+> Currently the hosted federation service is only available for enterprise. Other customers interested in hosted definitions should reach out to sales to discuss expanding their plans.
+
+## Uploading Service Definitions
+
+To begin, ensure all component services are up and running, and the working directory has a `.env` file containing the appropriate  `ENGINE_API_KEY` for your federated service. Then, upload each service to the gateway through the Apollo CLI:
+
+```bash
+apollo service:push       \
+  --serviceURL={{url}}    \
+  --endpoint={{url}}      \
+  --serviceName={{name}}  \
+  --federated
```

Comment:
Let's sync with the platform team on this, I don't think you need `--federated` anymore? I'm not sure the format we use in the rest of the docs for CLI variables, but let's be consistent one way or another